### DISCUSSION TITLE
_TZE204_x8fp01wi  not supported device

### DISCUSSION THREAD

**Discussion by AkLim94** [Koenkk/zigbee2mqtt#20155] [Devices]
Hi all,

I bought this device from AliExpress : https://pl.aliexpress.com/item/1005005679887461.html?spm=a2g0o.order_list.order_list_main.105.21ef1c24lwBKMz&gatewayAdapt=glo2pol

Unfortunately this device is not supported yet (with the 1.34.0) , so I found a discussion about this device:
https://github.com/Koenkk/zigbee2mqtt/issues/18750#issuecomment-1732396956

In the tuya.js file there's a the same model but with different fingerprint, 
I made a change by adding _TZE204_x8fp01wi in the fingerprint section
![image](https://github.com/Koenkk/zigbee2mqtt/assets/118060168/da40fc5c-a7fb-497f-99be-74ca2fe4a98d)

Now my device is supported and works well.

@Koenkk 
Is there's any chance to include it into next releases?

Thanks

Best regards

**Comment by Koenkk** [comment]
Done!

Changes will be available in the [dev branch](https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html) in a few hours from now.

**Reply by AkLim94** [reply]
Thanks a lot
Regards

**Reply by derkorte** [reply]
Hey Guys, when my home needs 500w and my PV system generates 1500w my consumption ist about 2000w instead of -1000w. Is there any way to give out negative values?

**Reply by derkorte** [reply]
@Koenkk have any idea to fix?

**Reply by cavefire** [reply]
I'm experiencing the same issue. Unfortunately, I couldn't find the sign in the packets, at least not in the data points listed in the converter. 
It appears that other data points are not being sent. If there is no configuration option available within the device itself (perhaps using the original Tuya app might reveal this), it seems that it might not be possible to resolve this.

**Reply by derkorte** [reply]
I've created a new issue for this... Hope it will get some attention 

https://github.com/Koenkk/zigbee2mqtt/issues/22639#issue-2296310557
